src/main/java/com/faforever/client/preferences/ui/SettingsController.java

```diff
@@ -255,9 +286,29 @@ private void configureThemeSelection(Preferences preferences) {
     );
   }
 
-  private void configureLanguageSelection() {
-    languageComboBox.setItems(FXCollections.singletonObservableList("English"));
-    languageComboBox.getSelectionModel().select(0);
+  private void configureLanguageSelection(Preferences preferences) {
+    languageComboBox.setItems(FXCollections.observableArrayList(LanguageInfo.getAllDisplayNames()));
```

Comment:
We want to set the `languageInfo` as an item and use a cell factory to generate what is displayed to the user (try to find similar code in the project)